test/test-tarantool/panic_on_broken_lsn.test.lua

```diff
@@ -0,0 +1,9 @@
+-- regression test for https://github.com/tarantool/tarantool-qa/issues/96
+
+ok, err = pcall(box.schema.user.grant, 'guest', 'replication')
```

Comment:
I would write a test like it is run in a clean tarantool instance (without `pcall()` in this case). Tests we add here are a kind of examples for others and it is better to fix the [problem](https://github.com/tarantool/test-run/issues/156) rather than suggest a workaround.

The rule of thumb is to revoke privileges we granted in the test. If we'll add `box.schema.user.revoke('guest', 'replication')` at the end, the test may be run several times on one instance (and we'll not need `pcall()`).

Reply:
removed pcall() wrapper and added command with rights revoke